rasa/core/utils.py

```diff
@@ -25,7 +25,7 @@
 from sanic.request import Request
 from sanic.views import CompositionView
 
-from rasa.core.constants import DEFAULT_REQUEST_TIMEOUT
+from rasa.utils.endpoints import EndpointConfig, read_endpoint_config
```

Comment:
```suggestion
from rasa.utils.endpoints import read_endpoint_config
```

unused import statement